gsd/pygsd.py

```diff
@@ -147,7 +144,7 @@ def __init__(self, file):
                 and self.__header.gsd_version != (0 << 16 | 3)):
             raise RuntimeError("Unsupported GSD file version: "
                                + str(self.__file))
-        if self.__header.gsd_version >= (2 << 16):
+        if self.__header.gsd_version >= (3 << 16):
```

Comment:
I'd recommend exposing the `gsd_make_version` function to Python rather than doing the bitshifts here again.

Reply:
`pygsd.py` is for use without the C library.

Reply:
Right, forgot about that.